.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Just out of curiosity: what does this line do?

Reply:
I used `create-react-app` to bootstrap the application because I didn't want to get us bogged down with web pack configurations. `create-react-app` causes an error with eslint version this is just to suppress the warning when we eject the application at a later stage we will remove this.